### ISSUE TITLE
[Feature Request] Make Masonry JS module available on frontend

### ISSUE THREAD

**Issue by fyayc-chrisat** [pojome/elementor#5537]
## Prerequisites

I have already contacted Elementor Pro support via email and they suggested to create an issue here.

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
I am using the plugin "Custom Skins for Elementor". The plugin allows to define a custom skin for the blog posts in the "Blog Post" Widget of Elementor Pro. However, when using this plugin, the "Masonry" effect does not work. I've digged into the source code of Elementor and I could code a solution for this if the module "elementor-utils/masonry" were available in the frontend of WP. In the backend, the module is already there in the `elementorFrontend.modules.Masonry` variable

**Describe the solution you'd like**
I can see in the source code of Elementor (https://github.com/pojome/elementor/blob/master/assets/dev/js/frontend/frontend.js, line 50) that the Masonry effect is loaded via require():
```
self.modules = {
		StretchElement: require( 'elementor-frontend/modules/stretch-element' ),
		Masonry: require( 'elementor-utils/masonry' )
};
```
If the Masonry module would also be available in the frontend, I could easily code a solution that makes the Masonry also work with custom skins. Is there any way I can get the Masonry module on the WP frontend for logged out users?

**Describe alternatives you've considered**
I tried achieving the same effect by using jQuery's Masonry plugin however it doesn't seem to fit well with Elementor and doesn't work. 

**Additional context**
In a nutshell:
- using "Custom Skins for Elementor" breaks the Masonry effect on custom skins
- making the Masonry module available in the WP frontend for logged out users as it is already available for backend users would make it easier to code a solution.

Thanks a lot. Since my client really wants custom skins as well as the Masonry effect, I'd be glad to point me to any direction that could possibly help.

**Comment by bainternet** [comment]
@fyayc-chrisat 

Its already available on the frontend:

```JS
elementorFrontend.modules.Masonry
```

thanks.

**Comment by fyayc-chrisat** [comment]
For all others stumbling across this. If you use custom skins, you need to run this snippet:
```
		var masonry = new elementorFrontend.modules.Masonry( {
			container: jQuery('.section-class-container'), //section css class of container (blog post widget)
			items: jQuery('.item'), //css class of each single item
			columnsCount: this.getSettings( 'colsCount' ), //column count
			verticalSpaceBetween: 0
		} );
```
